nixos/tests/sabnzbd.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }: {
+  name = "sabnzbd";
+  meta = with pkgs.lib; {
+    maintainers = with maintainers; [ jojosch ];
+  };
+
+  machine = { pkgs, ... }: {
+    services.sabnzbd = {
+      enable = true;
+    };
+
+    # unrar is unfree
+    nixpkgs.config.allowUnfreePredicate = pkg: builtins.elem (lib.getName pkg) [ "unrar" ];
```

Comment:
https://github.com/NixOS/nixpkgs/blob/1ffd57e92bdfed6936552b146d666604d0f3ba57/nixos/tests/nzbget.nix#L21
could the same thing be done here

Reply:
`unzip` is being inserted as part of `makeWrapper` in the `sabnzbd` package itself.